### DISCUSSION TITLE
Wifi on Pi 1.1 using Edimax USB dongle

### DISCUSSION THREAD

**Discussion by Roland-B** [rern/rAudio#259] [Fixed Issues] [answered]
Hi @rern,

I have attempted to get Pi1 working with Edimax USB dongle and I can scan for the wifi network and see it, but it never connects. Says "Done" when you attempt to connect but doesnt actually do anything. I have also had a friend try with a Pi1 and Edimax USB, and he has tried and gets exactly the same issue. 

Thanks in advance,

Roland B.

**Comment by R101G** [comment, marked as answer]
Is this discussion any help?

https://github.com/rern/rAudio-1/discussions/156

**Reply by Roland-B** [reply]
Hi @R101-Github 

I will give it a go and report back later, but I dont think it will work as the Pi 1 does not have any internal wireless so most people use an edimax dongle with a Pi 1, also noted that the access point does not work when booted on pi1 either and you HAVE to use LAN cable as the way to connect.  I also have used the Edimax USB dongle on Pi2 with no issue and did not have to disable using the script mentioned.

Thanks,

Roland B.

**Reply by R101G** [reply]
@Roland-B 

The Arch kernel does not seem to include the 8192cu driver which actually works, unlike Raspbian does. You need to install the driver and blacklist the dodgy ones, which the installer script does. You probably won't need to alter the boot files if there is no on-board wifi.

If you have used the same image on a Pi2, and it works, perhaps the 8192cu driver has now been restored to the kernel.

**Reply by Roland-B** [reply]
@R101-Github.

Brilliant Thanks for that, appreciated.

Roland B
